quantus/metrics/base.py

```diff
@@ -18,13 +18,28 @@
 from quantus.helpers import utils
 from quantus.helpers import warn
 from quantus.helpers.model.model_interface import ModelInterface
+from quantus.functions import postprocess_func
+from quantus.helpers.enums import ModelType, DataType, ScoreDirection, EvaluationCategory, EvaluationCategory
 
 
 class Metric:
     """
     Implementation of the base Metric class.
+
+    Attributes:
```

Comment:
Typically, attributes, which start with underscore, are supposed to be private, 
meaning, they should not part of the public API, and typically are left out of doc strings. Are you sure they should be mentioned in docstring?

Reply:
Hmm good point. But should we instead make them public directly? Why private actually...... this is useful information for the users.

Reply:
We can just skip the whole `@property def get_name`  ... by stating them as public attributes straight away.

Reply:
I'd say completely remove them, since they don't really make sense for base class. Moreover, because of the same name, the descriptors are already defined on the class level, which renders 
```python
@property
@abstractmethod
```
useless. I like the idea of abstract properties much more, since they will enforce each metric to implement them.

Reply:
Should be fixed now.